ЛР 1/ЛР 1.3.py

```diff
@@ -0,0 +1,12 @@
+# TODO Найдите количество книг, которое можно разместить на дискете
+disk = 1.44
+pages = 100
+rows = 50
+sumb = 25
+butes = 4
+books_in = butes*sumb*rows*pages
+disk_in = disk*1024*1024
+num_of_books = disk_in // books_in
+num_of_books2 = int(num_of_books)
+
+print("Количество книг, помещающихся на дискету:", num_of_books2)
```

Comment:
В данной ситуации можно сразу сделать значение целочисленным, не прибегая к созданию новой переменной.
```suggestion
num_of_books = int(disk_in // books_in)

print("Количество книг, помещающихся на дискету:", num_of_books)
```